docs/contracts/oracle-report-sanity-checker.md

```diff
@@ -3,6 +3,436 @@
 - [Source code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol)
 - [Deployed contract](https://etherscan.io/address/0x9305c1Dbfe22c12c66339184C0025d7006f0f1cC)
 
-OracleReportSanityChecker contains view methods to perform sanity checks of the Lido's oracle report and lever methods for granular
-tuning of the params of the checks. The contract limits the impact of incorrectly reported oracle data:
-if the report violates the restrictions enforced by the sanity checks, then the report tx reverts.
+Some vital data for the Lido protocol is collected off-chain and delivered on-chain via Oracle contracts:
+[`AccountingOracle`](./accounting-oracle.md), [`ValidatorsExitBusOracle`](./validators-exit-bus-oracle.md).
+Due to the high impact of data provided by the Oracles on the state of the protocol, each Oracle's
+report passes a set of onchain
+[sanity checks](https://en.wikipedia.org/wiki/Sanity_check).
+For the simplicity of the contracts responsible for handling Oracle's reports, all sanity checks were collected in the
+standalone `OracleReportSanityChecker` contract.
+
+Besides the validation methods, the `OracleReportSanityChecker` contract contains a [set of tunable limits and restrictions](#limits-list)
+used during the report validation process.
+To configure the limits values contract provides the lever methods described in the [standalone section](#lever-methods).
+Access to lever methods is restricted using the functionality of the
+[AccessControlEnumerable](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/utils/access/AccessControlEnumerable.sol)
+contract and a bunch of [granular roles](#permissions).
+
+## Limits List
+
+`OracleReportSanityChecker` introduces a new type `LimitsList` which contains all the limits used by the contract.
```

Comment:
Code piece with a structure would look better here, I believe.

Reply:
Added a code block with Solidity type, but I stayed the description in the current format. I think it's more readable than the comments in the code block.